source/_integrations/tasmota.markdown

```diff
@@ -29,18 +29,18 @@ This integration allows you to control [Tasmota](https://tasmota.github.io/docs/
 
 - MQTT server and the [MQTT integration](/integrations/mqtt/) set up in Home Assistant.
 - Tasmota devices flashed with version 9.2, or later.
-- Tasmota devices configured for native discovery (`SetOption19 0`)
+- Tasmota devices configured for native discovery (`SetOption19 1`)
```

Comment:
This is incorrect, it should be `SetOption19 0` for this integration.
See also: 


![image](https://user-images.githubusercontent.com/195327/112799112-34b4d280-906e-11eb-80b1-bdceb4f46749.png)

<https://tasmota.github.io/docs/Commands/>